File: csharp-dotnet2/src/main/CsharpDotNet2/IO/Swagger/Model/V3SiriStopsRefsDictionary.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;
using Newtonsoft.Json;

namespace IO.Swagger.Model {

  /// <summary>
  /// 
  /// </summary>
  [DataContract]
  public class V3SiriStopsRefsDictionary {
    /// <summary>
    /// Gets or Sets StopPointRefs
    /// </summary>
    [DataMember(Name="stop_point_refs", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "stop_point_refs")]
    public Dictionary<string, V3SiriReferenceDataDetail> StopPointRefs { get; set; }

    /// <summary>
    /// Gets or Sets UnmatchedStopPointRefs
    /// </summary>
    [DataMember(Name="unmatched_stop_point_refs", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "unmatched_stop_point_refs")]
    public Dictionary<string, string> UnmatchedStopPointRefs { get; set; }


    /// <summary>
    /// Get the string presentation of the object
    /// </summary>
    /// <returns>String presentation of the object</returns>
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class V3SiriStopsRefsDictionary {\n");
      sb.Append("  StopPointRefs: ").Append(StopPointRefs).Append("\n");
      sb.Append("  UnmatchedStopPointRefs: ").Append(UnmatchedStopPointRefs).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }

    /// <summary>
    /// Get the JSON string presentation of the object
    /// </summary>
    /// <returns>JSON string presentation of the object</returns>
    public string ToJson() {
      return JsonConvert.SerializeObject(this, Formatting.Indented);
    }

}
}

File: dart/lib/model/v3_siri_stops_refs_dictionary.dart
part of swagger.api;

class V3SiriStopsRefsDictionary {
  
  Map<String, V3SiriReferenceDataDetail> stopPointRefs = {};

  Map<String, String> unmatchedStopPointRefs = {};

  V3SiriStopsRefsDictionary();

  @override
  String toString() {
    return 'V3SiriStopsRefsDictionary[stopPointRefs=$stopPointRefs, unmatchedStopPointRefs=$unmatchedStopPointRefs, ]';
  }

  V3SiriStopsRefsDictionary.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
    stopPointRefs = V3SiriReferenceDataDetail.mapFromJson(json['stop_point_refs']);
    unmatchedStopPointRefs = json['unmatched_stop_point_refs'];
  }

  Map<String, dynamic> toJson() {
    return {
      'stop_point_refs': stopPointRefs,
      'unmatched_stop_point_refs': unmatchedStopPointRefs
     };
  }

  static List<V3SiriStopsRefsDictionary> listFromJson(List<dynamic> json) {
    return json == null ? new List<V3SiriStopsRefsDictionary>() : json.map((value) => new V3SiriStopsRefsDictionary.fromJson(value)).toList();
  }

  static Map<String, V3SiriStopsRefsDictionary> mapFromJson(Map<String, Map<String, dynamic>> json) {
    var map = new Map<String, V3SiriStopsRefsDictionary>();
    if (json != null && json.length > 0) {
      json.forEach((String key, Map<String, dynamic> value) => map[key] = new V3SiriStopsRefsDictionary.fromJson(value));
    }
    return map;
  }
}
